plugin.xml

```diff
@@ -7,7 +7,7 @@
     <description>ThaliPlugin to be used on top of JXcore</description>
     <license>MIT</license>
     <keywords>cordova,jxcore,node.js,thali</keywords>
-    <dependency id="io.jxcore.node" url="https://github.com/jxcore/jxcore-cordova" 
+    <dependency id="io.jxcore.node" url="../jxcore-cordova" 
```

Comment:
We should be getting jxcore-cordova from github not a parallel directory. Please revert.